R/zchunk_LB131.LV_R_GLU.R

```diff
@@ -29,42 +30,37 @@ module_aglu_LB131.LV_R_GLU_DISABLED <- function(command, ...) {
     iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
     L100.GTAP_LV_milUSD <- get_data(all_data, "L100.GTAP_LV_milUSD")
     L122.LC_bm2_R_HarvCropLand_Yh_GLU <- get_data(all_data, "temp-data-inject/L122.LC_bm2_R_HarvCropLand_Yh_GLU")
```

Comment:
...and if data coming in here with "X" years, strip them off here up top. When we remove temp data injection, you'll just have to remove a single line.